docusaurus.config.js

```diff
@@ -1,53 +1,53 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
```

Comment:
Hi @Siegfriedk, thanks you very much for your comments, I would like to have a chat with you next week whenever is possible (I'll text Max and request him to give me your MS because I am a guest at your org), I understand what you are saying about what the commit should not do, I rushed through this because there are some things discussed that I took for granted, so it's important to have a talk.